docs/4.Integrations/GitLab CLI.md

```diff
@@ -99,11 +95,6 @@ checkov:
     # Use `script` to emulate `tty` for colored output.
     - script -q -c 'checkov -d . ; echo $? > CKVEXIT'
     - exit $(cat CKVEXIT)
-  artifacts:
-    reports:
-      junit: "checkov.test.xml"
```

Comment:
why are we deleting the artifacts?

Reply:
I guess it isn't mandatory to fix the allow_failure issue. I'm removing it because it is not used as this script `checkov -d . ` outputs the console and not junit xml.

Reply:
can we keep it then and not delete it?

Reply:
Sounds good, restored.

Reply:
thanks!